### PR TITLE
Do not use babel in production

### PR DESCRIPTION
**Pull request by danielbayerlein**
It's not a good idea to use babel it in production. I created a build for production.

### CODE REVIEW

File: honeycomb-template/.config/server.development.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+const path = require('path');
+const webpackConfig = require('./webpack.config');
+const logConfig = require('./log');
+
+module.exports = {
+  connections: [
+    {
+      port: process.env.PORT || 3000,
+      routes: {
+        cors: true,
```
Comment (chrishelgert) [inline honeycomb-template/.config/server.development.js]: cors in production? do we really need this?

File: honeycomb-template/package.json [hunk 2]
```diff
@@ -21,13 +21,9 @@
   "author": "Daniel Bayerlein",
   "license": "MIT",
   "dependencies": {
-    "babel-core": "^6.9.1",
-    "babel-plugin-transform-class-properties": "^6.10.2",
-    "babel-plugin-transform-object-rest-spread": "^6.8.0",
-    "babel-plugin-transform-runtime": "^6.9.0",
-    "babel-preset-es2015": "^6.9.0",
-    "babel-preset-react": "^6.5.0",
+    "babel-runtime": "^6.11.6",
```
Comment (danielbayerlein) [inline honeycomb-template/package.json]: @chrishelgert see https://babeljs.io/docs/plugins/transform-runtime/

File: honeycomb-template/package.json [hunk 3]
```diff
@@ -2,12 +2,12 @@
...
   "description": "",
-  "main": "index.js",
+  "main": "src/server/server.js",
   "scripts": {
-    "build": "npm run build:development",
-    "build:development": "cross-env NODE_ENV=development webpack --config .config/webpack.config.js",
-    "build:production": "cross-env NODE_ENV=production webpack --config .config/webpack.config.js",
-    "clean": "rimraf pids logs coverage .nyc_output public/javascripts/*.bundle.js*",
+    "build": "npm run build:babel && npm run build:webpack",
+    "build:babel": "cross-env BABEL_ENV=production babel src/ --out-dir dist/ --source-maps --only src/server/",
```
Comment (danielbayerlein) [inline honeycomb-template/package.json]: @chrishelgert It's 👌.

File: honeycomb-template/.config/server.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+          },
+          path: 'views',
+          relativeTo: path.resolve(__dirname, '../dist/server'),
+        },
+      },
+    },
+  ],
+};
+
+module.exports = Hoek.merge(
```
Comment (danielbayerlein) [inline honeycomb-template/.config/server.js]: @chrishelgert One single configuration file for the server.

File: honeycomb-template/.config/server.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
+const path = require('path');
+const logConfig = require('./log');
+const webpackConfig = require('./webpack.config');
+const Hoek = require('hoek');
+
+const defaultConfig = {
+  connections: [
+    {
+      port: process.env.PORT || 3000,
```
Comment (danielbayerlein) [inline honeycomb-template/.config/server.js]: @chrishelgert We don't need `router.cors`

**Comment by danielbayerlein** [comment]
@chrishelgert I've edited the points from the review.

**Pull request merged**